docusaurus.config.ts

```diff
@@ -0,0 +1,141 @@
+import { themes as prismThemes } from "prism-react-renderer";
```

Comment:
It might be worth refining a few of the settings here as it's got some non-mercurius references in this file?

Reply:
Yes, this import is for the colors of the theme that is invoked at the end of the `docusaurus.config.ts` file :

```
prism: {
      theme: prismThemes.github,
      darkTheme: prismThemes.oceanicNext
}
```

Reply:
Sorry, I meant the rest of `docusaurus.config.ts` file below - I'll leave comments on the bits I mean to be more explicit :)